README.md

```diff
@@ -258,6 +258,22 @@ module "api_gateway" {
 | redeployment_trigger                      | string       | A unique string to force a redeploy of the api gateway. If not set manually, the module will use the configurations for endpoints, api_key, and authorizer config to decide if a redeployment is necessary. | (yes)    |                                                           |
 | tags                                      | map(string)  | Map of tags to assign to the Lambda Function and related resources                                                                                                                                          | no       | `{}`                                                      |
 
+#### setup-tfstate
+
+```hcl
+module "setup_tfstate" {
+  source = "github.com/THEY-Consulting/they-terraform//aws/setup-tfstate"
+
```

Comment:
QoL: Perhaps we should give the ability to specify the region where the s3 state bucket will be deployed to? 🤔 Thinking of data privacy concerns/criterias one could have to satisfy.

Reply:
the bucket location depends on the provider region that is setup outside the module. I added a short info in the README